lib/rust/ensogl/core/src/timer/delayed_repeats.rs

```diff
@@ -0,0 +1,46 @@
+//! Delayed repeating timer based on browser timer APIs.
+
+use super::Interval;
+use super::Timeout;
+use crate::prelude::*;
+
+use std::time::Duration;
+
```

Comment:
No sections. Please read Enso Style Guide and fix all the related places in the code.